DigitalLearningSolutions.Data/Models/CourseCompletion/CourseCompletion.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace DigitalLearningSolutions.Data.Models.CourseCompletion
+{
+    using System;
+
+    public class CourseCompletion
+    {
+        public int Id { get; }
+        public string ApplicationName { get; }
+        public string CustomisationName { get; }
+        public bool IncludeCertification { get; }
+        public DateTime? Completed { get; }
+        public DateTime? Evaluated { get; }
+        public int MaxPostLearningAssessmentAttempts { get; }
+        public bool IsAssessed { get; }
+        public int PostLearningAssessmentPassThreshold { get; }
+        public int DiagnosticAssessmentCompletionThreshold { get; }
+        public int TutorialsCompletionThreshold { get; }
+        public int? DiagnosticScore { get; }
+        public int? LearningDone { get; }

```

Comment:
What does this field represent? 🙂

Reply:
`DiagnosticAssessmentCompletionThreshold` corresponds to `DiagCompletionThreshold` from the old stored procedure, and will be used to produce text like "To complete this course, you must achieve {DiagnosticAssessmentCompletionThreshold}% in the diagnostic assessment" 🙂

Reply:
Sorry I meant the line above `LearningDone` 🙂

Reply:
Ah this is the name of the same field from the old stored procedure, it essentially is the percentage of tutorials completed. I could change the name to something clearer, maybe `TutorialsCompleted`? 🙂

Reply:
Yes I think even `PercentageTutorialsCompleted`, to be completely clear 🙂

Reply:
Renamed in 64314db 🙂